test/resources/microshift-process.resource

```diff
@@ -80,6 +77,16 @@ Restart MicroShift
     [Documentation]    Restart the MicroShift service
     ${ushift_pid}=    MicroShift Process ID
     ${ushift_etcd_pid}=    MicroShift Etcd Process ID
+    # Force delete of core pods to avoid extended downtime after a restart.
```

Comment:
While you are right, there are some details that are specific to the tests, like performing many restarts in a short period of time. While this is also possible in a real deployment, its unlikely to happen as it may cause disruptions to applications, not just the core components. If you wait just a bit more it will heal by itself, but tests dont allow that much waiting.
Anyhow, what I am after now is the 'forbidden' logs from a different test, which is unexpected. I will try again with the TIMESTAMP env var.
If this doesnt fix it then I think we are better off with skipping these tests until its ok again.

Reply:
> While you are right, there are some details that are specific to the tests, like performing many restarts in a short period of time. While this is also possible in a real deployment, it's unlikely to happen as it may cause disruptions to applications, not just the core components. If you wait just a bit more it will heal by itself, but tests dont allow that much waiting.

Should the tests wait for longer?

> Anyhow, what I am after now is the 'forbidden' logs from a different test, which is unexpected. 

What is being forbidden?

> I will try again with the TIMESTAMP env var. If this doesnt fix it then I think we are better off with skipping these tests until we solve it again.